### ISSUE TITLE
Cannot load 1.7.0b

### ISSUE THREAD

**Issue by awshirley** [akasma74/Hass-Custom-Alarm#21]
Alarm Version 1.7.0b

**Platform & Browser:**
CentOS 7 and Home Assistant 0.95.1

Browser
Chrome  75.0.3770.100
 - Browser name & version

**Describe the bug**
HA cannot run Alarm 1.7.0b after installation.  No error messages in HA log, other than the standard warning about custom components.  When checking the config with the config check button in configuration menu it shows this:

Configuration invalid
Platform not found: alarm_control_panel.bwalarm

**To Reproduce**
I removed all of the 1.6.0b directories from custom_components and alarm.yaml from the config directory.  I installed 1.7.0b into custom_components/bwalarm (including all sub-folders), resources into config directory.  Configuration.yaml was changed to
alarm_control_panel: !include resources/bwalarm/bwalarm.yaml, restart HA.  Use the check config from the configuration menu.

**Expected behavior**
To see alarm on ingress menu.

**Comment by akasma74** [comment]
I see, the logging feature is broken in 0.94.2+ so you don't see anything.
Anyway, could you download and install [this](https://github.com/akasma74/Hass-Custom-Alarm/releases/tag/v1.7.0b) release?

**Comment by onkytonk** [comment]
Experiencing the same issue as @awshirley 
Installed 1.7.0b and getting the same config invalid message. 
I was previously on 1.6.0b and removed all traces.
Added the resource folder, and bwalarm location in config.yaml as per instructions.
Currently on HA 0.95.4

**Comment by akasma74** [comment]
@onkytonk could you download, install and test [this](https://github.com/akasma74/Hass-Custom-Alarm/releases/tag/v1.7.0b) release?

**Comment by onkytonk** [comment]
Tried the release you linked to. 

Same issue. 

Log: "You are using a custom integration for bwalarm which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you do experience issues with Home Assistant."

That's it. 

No Alarm Panel on the sidebar and alarm is missing from entiry states.

**Comment by akasma74** [comment]
Can you check if you have `const.py` file in your `custom_components/bwalarm/` folder?
And can you post output of
```
hassio ha logs
```
?

**Comment by onkytonk** [comment]
> Can you check if you have `const.py` file in your `custom_components/bwalarm/` folder?
> And can you post output of
> 
> ```
> hassio ha logs
> ```
> 
> ?

Definitely no const.py in the bwalarm folder. This isn't in the zip file either - should it be?

Here's the full log:
```
2019-07-01 18:17:01 WARNING (MainThread) [homeassistant.loader] You are using a custom integration for bwalarm which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you do experience issues with Home Assistant.
2019-07-01 19:58:38 ERROR (MainThread) [homeassistant.components.hassio] Platform not found: alarm_control_panel.bwalarm
2019-07-01 19:59:11 ERROR (MainThread) [homeassistant.components.hassio] Platform not found: alarm_control_panel.bwalarm
```

**Comment by akasma74** [comment]
Yeah.. It's Github or my hands..
Try [this](https://github.com/akasma74/Hass-Custom-Alarm/releases/tag/v1.7.1b) one, it should work this time ;)

**Comment by onkytonk** [comment]
> @onkytonk could you download, install and test [this](https://github.com/akasma74/Hass-Custom-Alarm/releases/tag/v1.7.0b) release?

Looks like we're back in action! Thanks mate. :)

**Comment by akasma74** [comment]
I'll close this issue as it seems to be resolved in v.1.7.1b

**Comment by wes1993** [comment]
Hello, i have installed manually with Hassio 0.97.1 and i have the same error, how can i solve the problem? i have downloaded all files from the github repo...
Thanks  a Lot
Stefano

**Comment by akasma74** [comment]
please make sure you followed all the installation steps as it was one-off issue and it's fixed now.
if not, please open a new issue.

**Comment by wes1993** [comment]
Thanks a lot Akasma, i have solved, i have downloaded the 1.10.1 version!! :-D
